MOCC/src/configurations/PlanActorsController/src/DatabaseHelper.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import division, print_function
+from plan_actor_schema import PlanActors
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+
+# Connecting to the database
+engine = create_engine('sqlite:///plan_actors.db', echo=False)
+Session = sessionmaker(bind=engine)
+
+# Create session
+s = Session()
+
+
+# To add the plan actor fields into the database
+def add(data):
+    address = data[0]  # Address
+    type_t = data[1]  # Type
+    avail_start = data[2]  # available start date time
+    avail_end = data[3]  # available end date time
+    capabilities = data[4]  # capability string - JSON
+
+    query = s.query(PlanActors).filter(PlanActors.address.in_([address]))
+    result = query.first()
```

Comment:
please put inside a try-except-finally block.
as it is, if there's an exception, there'll be a non-commited transaction left open